presto-main/etc/config.properties

```diff
@@ -41,3 +41,4 @@ plugin.bundles=\
 presto.version=testversion
 distributed-joins-enabled=true
 node-scheduler.include-coordinator=true
+experimental.iterative-optimizer-timeout=60s
```

Comment:
I thought this would be the example config where you look first when configuring Presto.
But with https://github.com/prestodb/presto/pull/7458#discussion_r105232529, we probably don't need to change that configuration.
I'll address https://github.com/prestodb/presto/pull/7458#discussion_r105232529 and discard this commit.